vic/vic_run/src/CalcBlowingSnow.c

```diff
@@ -271,18 +271,18 @@ CalcBlowingSnow(double   Dt,
  *****************************************************************************/
 double
 qromb(double (*funcd)(),
-      double es,
-      double Wind,
-      double AirDens,
-      double ZO,
-      double EactAir,
-      double F,
-      double hsalt,
-      double phi_r,
-      double ushear,
-      double Zrh,
-      double a,
-      double b)
+      double   es,
```

Comment:
I assume this is from `uncrustify`. Slightly odd that it adds spaces. Probably trying to match `*funcd`. Just leave as is.